typescript/packages/well-log-viewer/src/components/WellLogView.tsx

```diff
@@ -1036,9 +1038,16 @@ export interface WellLogViewOptions {
 
 export interface WellLogViewProps {
     /**
-     * Object from JSON file describing single well log data.
+     * Object from JSON file describing one or more sets of well log data.
+     * @depreacted Use `wellLogSets` instead
```

Comment:
Spelling, should be "deprecated"

Reply:
Personally, I don't bother to recopy doc. I only write the @deprecated part. ;)